### DISCUSSION TITLE
WOL 重连出现  隧道启动失败: 每个节点只能同时启动一个 WOL 隧道

### DISCUSSION THREAD

**Discussion by RrrrLin** [natfrp/wiki#445] [使用问题] [answered]
版本 3.0.0 / 3.0.9 / 0.51.0-sakura-3
环境 Linux ARMv7
软件只有一条WOL隧道配置

2024/02/21 08:11:14 I Tunnel/HOME_PC [22/210645/d1fa] 隧道启动成功
2024/02/21 15:05:50 I Tunnel/HOME_PC [22/210645/d1fa] 开始无缝重连
2024/02/21 15:06:00 E Tunnel/HOME_PC [22/210645/d1fa] 无缝重连失败: timeout
2024/02/21 15:06:00 I Tunnel/HOME_PC [22/210645/d1fa] 访问隧道管理器正常退出
2024/02/21 15:06:00 I Tunnel/HOME_PC [22/210645/d1fa] 工作线程正常退出
2024/02/21 15:06:00 E Tunnel/HOME_PC [22/210645/d1fa] 网络波动导致连接断开 (EOF), 正在重试
2024/02/21 15:06:00 I Tunnel/HOME_PC [22/210645/d1fa] 正在重试连接节点...
2024/02/21 15:06:01 I Tunnel/HOME_PC [22/210645/4444] 连接节点成功, 运行 ID [210645-4444086d]
2024/02/21 15:06:01 I Tunnel/HOME_PC [22/210645/4444] 重试成功
2024/02/21 15:06:01 I Tunnel/HOME_PC [22/210645/4444] 隧道启动中: [HOME_PC, wol]
2024/02/21 15:06:01 I Tunnel/HOME_PC [22/210645/4444] 限速已更新: 10 Mibit/s
2024/02/21 15:06:01 W Tunnel/HOME_PC [22/210645/4444] 隧道启动失败: 每个节点只能同时启动一个 WOL 隧道
2024/02/21 15:06:34 W Tunnel/HOME_PC [22/210645/4444] 隧道启动失败: 每个节点只能同时启动一个 WOL 隧道
2024/02/21 15:07:07 W Tunnel/HOME_PC [22/210645/4444] 隧道启动失败: 每个节点只能同时启动一个 WOL 隧道
2024/02/21 15:07:40 W Tunnel/HOME_PC [22/210645/4444] 隧道启动失败: 每个节点只能同时启动一个 WOL 隧道

**Comment by fakeboboliu** [comment]
感谢您的反馈，这是一个日志显示错误，WOL 此时应当可以正常使用。
我们会在下个版本中修复这个问题。

**Reply by RrrrLin** [reply]
根据我的使用情况，这时候远程开机 WOL隧道并没有在线。希望在修复时做额外的排查。感谢。

**Reply by fakeboboliu** [reply, marked as answer]
> 根据我的使用情况，这时候远程开机 WOL隧道并没有在线。

您是正确的，这是因为一个在线判断机制的问题。
因为一直展示离线从而在前端无法选中使用。

在我们发布修复前，您应当可以在启动超时后自动重启时恢复 WOL 状态。
我们也推荐您启用远程管理以保障可用性冗余。
